index.js

```diff
@@ -27,14 +27,13 @@ var formatOutput = function(report, file, options) {
     return err;
   });
 
-  var output = {
+  return {
```

Comment:
I'm thinking this needs to go. Maybe move the formatting you've created in this module to it's own module, and register it on the official way? That way, much of this mapping logic goes away.